### ISSUE TITLE
a11y: Logo alt is overridden by aria-label

### ISSUE THREAD

**Issue by iamjolly** [18F/all_sorns#145]
## Description:
The logo on the site has an image of text which reads SORNS DASH, however, the link that wraps is has `aria-label="home"` set to it. This replaces the `alt` attribute which is set on the image so that only "home" is announced by screen readers. This causes a failure of WCAG SC 1.1.1 Non-text Content (Level A).

## Screenshot:
![screenshot of how the logo gets announced as home](https://user-images.githubusercontent.com/1093423/105549057-b2024680-5cbd-11eb-8d27-83ba68dc7a6d.png)

## WCAG SC
[Success Criterion 1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content): All non-text content that is presented to the user has a text alternative that serves the equivalent purpose

## Recommended fix:
1. Remove the `aria-label` from the anchor around the logo.
2. Remove the word "logo" from the `alt` on the `img` of the logo. Consider setting the `alt` value to something like "SORNS DASH - Home" if you still want to give a hint about where the link will go.
